package.json

```diff
@@ -31,5 +31,8 @@
     "lerna": "^6.5.1",
     "npm-run-all": "^4.1.5",
     "tsc": "^2.0.3"
+  },
+  "dependencies": {
+    "@walletconnect/did-jwt": "^2.0.0"
```

Comment:
Hope I'm not missing something but why are we adding a dependency at the root level?

Reply:
Think it’s accidental 👍

cc @Cali93